Core/OxidEeEvents.php

```diff
@@ -32,6 +33,7 @@ class OxidEeEvents
     const ORDER_TABLE = "oxorder";
     const PAYMENT_TABLE = "oxpayments";
     const TRANSACTION_TABLE = "wdoxidee_ordertransactions";
+    const PAYMENT_METADATA_TABLE = "wdoxidee_oxpayments_metadata";
```

Comment:
You save here configurations of are payment methods, if i understand correctly. If so can we please name it in that context, something like 'payment method settings'?

Reply:
We chose metadata as a term to describe additional information attached to a specific model – in this case the payment model. This is a common term also used in other frameworks, e.g. in WordPress.

Reply:
ok, fine with me.